src/components/Modal/BaseModal.tsx

```diff
@@ -251,14 +252,18 @@ function BaseModal(
                     avoidKeyboard={avoidKeyboard}
                     customBackdrop={shouldUseCustomBackdrop ? <Overlay onPress={handleBackdropPress} /> : undefined}
                 >
-                    <ModalContent onDismiss={handleDismissModal}>
-                        <View
-                            style={[styles.defaultModalContainer, modalPaddingStyles, modalContainerStyle, !isVisible && styles.pointerEventsNone]}
-                            ref={ref}
-                        >
-                            <ColorSchemeWrapper>{children}</ColorSchemeWrapper>
+                    <FocusTrapForModal active={isVisible}>
+                        <View>
```

Comment:
Is this view necessary?

Reply:
Yes, it's necessary because `FocusTrap` creates `ref` on it's first child. Without this `View` there's error saying that `function components cannot be given refs`.  It's safer to add this `View` rather than to try to forward refs through `ModalContent`

Reply:
Sounds good, let's add a comment explaining why the view is necessary.

Reply:
Is this `<View>` component really necessary? The one wrapping the `<ModalContent>`  

(image after removing)
<img width="469" alt="image" src="https://github.com/Expensify/App/assets/67908363/9f31766d-f69f-46b9-a408-0e100636ec19">
What do you think @fedirjh?

Reply:
nvm I found the thread with explanation 😄